### ISSUE TITLE
Activate Visual Editor by default (PERSOID)

### ISSUE THREAD

**Issue by FantasticoFox** [inblockio/mediawiki-extensions-Aqua#242]
The Visual Editor will fail to load with additional settings being configured to allow local access to the required API on the MediaWiki Server.

The apache needs to be configured to allow for localhost access for the Visual Editor.

- Please document what changes where required so we are aware for future packaging and more importantly for 'required open API slots' in the hardening process

As this is a massive UX improvement it is important for the user-expierence of our software.

**Comment by mrglaser** [comment]
VisualEditor needs to be enabled in LocalSettings.php like this:
```
wfLoadExtension( 'VisualEditor' );
wfLoadExtension( 'Parsoid', "$IP/vendor/wikimedia/parsoid/extension.json" );
$wgDefaultUserOptions['visualeditor-enable'] = 1;
$wgDefaultUserOptions['visualeditor-newwikitext'] = 1;
#$wgDefaultUserOptions['visualeditor-autodisable'] = true;

$wgVirtualRestConfig['modules']['parsoid'] = array(
    'url' => 'http://localhost' . $wgScriptPath . '/rest.php',
);
```
The reason it did not work before is that Parsoid is accessed only from inside the docker container. This means, it does not use Port 9352, but standard port 80 instead. This is set in the `$wgVirtualRestConfig` in the snipped above.
In the current setup, you have set the option 'visualeditor-autodisable'. This needs to be commented out.

**Comment by it-spiderman** [comment]
Should i add this config to https://github.com/inblockio/aqua-PKC/blob/main/aqua/extraAquaSettings.php?

**Comment by FantasticoFox** [comment]
FIXED in 4c8098bd6cbd3c171adf2486e91fd2d94e31b8d9 and working on localhost.
